tests/test_configurer.py

```diff
@@ -0,0 +1,36 @@
+from cdci_data_analysis.configurer import DataServerConf
+import os
+import pytest
+
+def test_data_server_conf(tmpdir, caplog):
+    #required_keys = ['data_server_url']
+    good_dict = {'data_server_url': 'https://data-server:5000',
+                 'dispatcher_mnt_point': 'data',
+                 'data_server_remote_cache': 'reduced/ddcache',
+                 'dummy_cache': 'data/dummy_prods'
+                 }
+    bad_dict = {'data_server_url': 'https://data-server:5000',
+                 'data_server_port': '5000',
+                 'dispatcher_mnt_point': 'data',
+                 'data_server_cache': 'reduced/ddcache',
+                 'dummy_cache': 'data/dummy_prods'
+                 }
+    ugly_dict = {'data_server_host': 'data-server',
```

Comment:
nice! but if we provide data_server_host and data_server_url it will say nothing of the wrong value, right?